index.html

```diff
@@ -61,8 +61,6 @@ <h2 class="prof">I’m a Software Developer</h2>
     <h3 class="works-h3">My Recent Work</h3>
     <hr class="works-hr">
   </div>
-  
-
 
 <div class="box-container">
```

Comment:
- Your works section should be generated dynamically using JavaScript, which means you would not have any hard coded html. currently you still have html code which shouldn't be the case i kindly suggest you work on creating your works section dynamically :+1: 
![goof](https://github.com/Xghayor/Portfolio/assets/89055502/dcb9877e-86df-4be5-876a-67bf4118cc36)